tests/functional/adapter/store_failures/fixtures.py

```diff
@@ -0,0 +1,48 @@
+seed_csv = """
+id,value
+1,1
+2,2
+3,3
+4,4
+""".lstrip()
+
+table_model = """
+{{config(materialized = "table")}}
+select * from {{ ref('seed') }}
+"""
+
+table_profile_yml = """
+version: 2
+models:
+  - name: table_model
+    description: only one model
```

Comment:
Strip `description` from the model and the columns if they are not relevant

Reply:
done